src/ref-field/ref-field.component.ts

```diff
@@ -100,6 +100,10 @@ export class RefFieldComponent implements OnChanges {
     return this.appGlobalsService.templates[this.refConfig.templateName];
   }
 
+  get pathStringWithRef(): string {
+    return this.pathString + '/$ref';
```

Comment:
You should use `PathUtilService` for this.

Reply:
interesting 😄